src/zoid/qr-code/component.jsx

```diff
@@ -33,15 +32,52 @@ export function getQRCodeComponent() : QRCodeComponent {
                 height: '100%'
             },
             logger:            getLogger(),
-            prerenderTemplate: ({ doc, props }) => {
+            prerenderTemplate: ({ doc, props, close }) => {
+                const style = `
+                #close {
+                    position: absolute;
+                    right: 16px;
+                    top: 16px;
+                    width: 16px;
+                    height: 0;
+                    opacity: 0.6;
+                    z-index: 10;
+                    padding: 0;
+                    border: none;
+                    cursor: pointer;
```

Comment:
I think `<button>` uses a pointer by default. Can you check? We should keep these styles down to the bare minimum.
```suggestion
```